pages/linux/gpasswd.md

```diff
@@ -0,0 +1,23 @@
+# gpasswd
+
+> Administer "/etc/group" and "/etc/gshadow". Administer linux group.
```

Comment:
Is the second line necessary ? The command only manages those 2 files. I think "Administer "/etc/group" and "/etc/gshadhow" files" sounds clear enough.

Reply:
I agree.